features/admin.push-providers.v1/constants/push-provider-constants.ts

```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { ExtensionTemplateCategoryInterface } from "@wso2is/admin.template-core.v1/models/templates";
+
+export class PushProviderConstants {
+
+    public static readonly NOTIFICATION_PROVIDER_CATEGORIES_INFO: ExtensionTemplateCategoryInterface[] = [
```

Comment:
Do we need an array here since we have only one category at the moment?

Reply:
This is passed to extension templates provider and it expects the categories to be passed as an array[1].

[1] https://github.com/wso2/identity-apps/blob/master/features/admin.template-core.v1/provider/extension-templates-provider.tsx#L50